src/multio/action/statistics/Statistics.cc

```diff
@@ -332,6 +333,16 @@ void Statistics::executeImpl(message::Message msg) {
         stat = fieldStats_.find(key);
     }
 
+    // If the action sets the option that initial data is supposed to be passed to the next action in case of several statistic actions, 
+    // AND the model sends initial data AND the internal tracker if any message has been directly passed through is still false the message is passed through to the next action
+    // TODO check if this is a good position for this check. It looks like the next chekc would filter this message out anyway. 
+    //std::cout<<"STEBA opt_.solver_send_initial_condition() " << opt_.solver_send_initial_condition() << " opt_.initialDataPassThrough() " << opt_.initialDataPassThrough() <<" !initialDataAlreadyPassedThrough_ " << !initialDataAlreadyPassedThrough_ << std::endl;
+    if(opt_.solver_send_initial_condition() &&  opt_.initialDataPassThrough() && !initialDataAlreadyPassedThrough_) {
```

Comment:
So this assumes - without checking the metadata - that the 
user properly configured the action to what the model is sending.

Is it possible to assert some things? I.e. a pass through message has step=0 ?